hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java

```diff
@@ -158,5 +167,18 @@ public void enrichTrace(StructuredTrace trace) {
               EnrichedSpanConstants.getValue(ErrorMetrics.ERROR_METRICS_TOTAL_SPANS_WITH_ERRORS),
               MetricValueCreator.create(errorCount));
     }
+
+    Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+
+    ApiTraceGraph apiTraceGraph = new ApiTraceGraph(trace);
+    for (ApiNode<Event> apiNode : apiTraceGraph.getApiNodeList()) {
+      Optional<Event> entryEvent = apiNode.getEntryApiBoundaryEvent();
+      int apiTraceErrorCount =
+          (int) apiNode.getEvents().stream().filter(this::findIfEventHasError).count();
+      if (entryEvent.isPresent()) {
+        enrichedAttributes.put(
+            API_TRACE_ERROR_COUNT, AttributeValueCreator.create(apiTraceErrorCount));
```

Comment:
The attribute name is a big confusing. To me it sounds like, it's counting the total number of errors in the api node, but it's actually just counting the number of events that had error

May be `API_TRACE_ERROR_SPAN_COUNT`? or, whatever seems more explanatory